src/main/java/org/opensearch/ad/rest/RestValidateAnomalyDetectorAction.java

```diff
@@ -59,31 +62,19 @@ public String getName() {
 
     @Override
     public List<Route> routes() {
-        return ImmutableList.of(
-                new Route(RestRequest.Method.POST,
-                        String.format(Locale.ROOT, "%s/%s", AnomalyDetectorPlugin.AD_BASE_DETECTORS_URI, VALIDATE)),
-                new Route(RestRequest.Method.POST,
-                        String.format(Locale.ROOT, "%s/%s/{%s}", AnomalyDetectorPlugin.AD_BASE_DETECTORS_URI, VALIDATE, TYPE))
-        );
+        return ImmutableList
+            .of(
+                new Route(
+                    RestRequest.Method.POST,
+                    String.format(Locale.ROOT, "%s/%s", AnomalyDetectorPlugin.AD_BASE_DETECTORS_URI, VALIDATE)
```

Comment:
If user use this URL without inputing the TYPE, will we just detect blocking issues?

Reply:
Yes it defaults to blocking issues only